### PR TITLE
Service Discovery and Nginx

### PR DESCRIPTION
**Pull request by rkipp1210**
Adds nginx, consul and registrator to the project

### CODE REVIEW

**Comment by emches** [comment]
Cleared out old test files and reorganized directories

File: swarm_setup/consul_demo/clean_all.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+docker service rm nginx_proxy
+docker service rm model1
+docker service rm model2
+docker service rm registrator-swarm-manager
+docker service rm registrator-swarm-worker1
+docker service rm registrator-swarm-worker2
+
+for host in ${hosts[@]}; do
+  eval $(docker-machine env ${host})
+  docker rm -f $(docker ps -a -q)
```
Comment (bhaskaraspb) [inline swarm_setup/consul_demo/clean_all.sh]: This is so useful 😄

Comment (emches) [reply swarm_setup/consul_demo/clean_all.sh]: 💁

File: swarm_setup/multi_master_swarm_init.sh [hunk 2]
```diff
@@ -1,61 +0,0 @@
-#!/bin/bash
```
Comment (rkipp1210) [inline swarm_setup/multi_master_swarm_init.sh]: did you mean to delete this file? I think this is @bhaskaraspb's multi master POC file

Comment (bhaskaraspb) [reply swarm_setup/multi_master_swarm_init.sh]: yeah, can we add that file back in? 💁‍♂️  it's there for testing purposes.

**Comment by emches** [comment]
Added back the multi master stuff!

**Pull request merged**